frontend/src/employee-mobile-frontend/child-attendance/ChildListItem.tsx

```diff
@@ -188,6 +191,13 @@ export default React.memo(function ChildListItem({
                   />
                 </Link>
               ) : null}
+              <RoundIcon
+                content={`${childAge}v`}
+                color={
+                  childAge < 3 ? colors.accents.a6turquoise : colors.main.m1
+                }
+                size="m-age"
+              />
```

Comment:
You can do it like this instead:

```ts
<AgeRoundIcon ... size="m" />
```

Bottom of the file:
```ts
const AgeRoundIcon = styled(RoundIcon)`
  &.m {
    font-size: 14px;
  }
```

The `&.m` part works around CSS selector precedence.

Reply:
Age icon font size related styling extracted from base component